store/cart.php

```diff
@@ -26,6 +26,8 @@
     if (count($cart) > 0) {
 ?>
 
+    <script>ga('send', 'event', 'Store', 'Cart Visit')</script>
```

Comment:
It is on most of the others, I'm not sure why when we start with a `div` we indent immediately, but not with a `form`.